README.md

```diff
@@ -1 +1,56 @@
-# Vehicles-recipe
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# Vehicles App
+
+<div style="display: flex">
+    <img src="./public/mobile-black(1).png" alt="" style="width: 40%"/>
+    <span style="width: 15%"></span>
+    <img src="./public/mobile-black(2).png" alt="" style="width: 40%"/>
+</div>
```

Comment:
- [OPTIONAL]  Please, add a meaningful  `Project description`. Remember A project description is a high-level overview of why you're doing a project. The document explains a project's objectives and essential qualities. Is an essential document that provides your audience with a comprehensive overview of the project. Think of it as the elevator pitch that focuses on what and why without delving into how. 😉 💯 

- The README file It is the first file a person will see when they encounter your project, so it should be fairly brief but detailed. It will make your project stand out from a bunch of others.